app/_src/gateway/plugin-development/wasm/filter-configuration.md

```diff
@@ -0,0 +1,226 @@
+>     {
+>       "name": "my-filter",
+>       "config": {
+>         "my-property": "My value"
+>       }
+>     }
+>   ]
+> }
+> EOF
+```
+
+### Example
+
+In this example, `set-response-header` is a filter that sets a single response
+header. It takes a JSON configuration with a header name and value.
+
+Here is the contents of `$KONG_WASM_FILTERS_PATH/set-response-header.meta.json`:
+
+```json
+{
+  "config_schema": {
+    "type": "object",
+    "properties": {
+      "header-name": {
+        "type": "string"
+      },
+      "header-value": {
+        "type": "string"
+      }
+    },
+    "required": [
+      "header-name",
+      "header-value"
+    ]
+  }
+}
+```
+
+With the schema in place, configurations that do not pass validation are
+rejected:
+
+```console
+$ cat <<'EOF' | http POST :8001/services/test/filter-chains
+> {
+>   "name": "my-filter-chain",
+>   "filters": [
+>     {
+>       "name": "set-response-header",
+>       "config": {
+>         "header-name": "X-My-Custom-Header"
+>       }
+>     }
+>   ]
+> }
+> EOF
+HTTP/1.1 400 Bad Request
+Access-Control-Allow-Credentials: true
+Access-Control-Allow-Origin: *
+Connection: keep-alive
+Content-Length: 203
+Content-Type: application/json; charset=utf-8
+Date: Mon, 06 Nov 2023 00:11:33 GMT
+Server: kong/3.6.0
+X-Kong-Admin-Latency: 6
+
+{
+    "code": 2,
+    "fields": {
+        "filters": [
+            {
+                "config": "property header-value is required"
+            }
+        ]
+    },
+    "message": "schema violation (filters.1: {\n  config = \"property header-value is required\"\n})",
+    "name": "schema violation"
+}
+```
+
+Whereas valid configurations are accepted:
+
+```console
+$ cat <<'EOF' | http POST :8001/services/test/filter-chains
+> {
+>   "name": "my-filter-chain",
+>   "filters": [
+>     {
+>       "name": "set-response-header",
+>       "config": {
+>         "header-name": "X-My-Custom-Header",
+>         "header-value": "Hello, Wasm!"
+>       }
+>     }
+>   ]
+> }
+> EOF
+HTTP/1.1 201 Created
+Access-Control-Allow-Credentials: true
+Access-Control-Allow-Origin: *
+Connection: keep-alive
+Content-Length: 348
+Content-Type: application/json; charset=utf-8
+Date: Mon, 06 Nov 2023 00:21:30 GMT
+Server: kong/3.6.0
+X-Kong-Admin-Latency: 23
+
+{
+    "created_at": 1699230090,
+    "enabled": true,
+    "filters": [
+        {
+            "config": {
+                "header-name": "X-My-Custom-Header",
+                "header-value": "Hello, Wasm!"
+            },
+            "enabled": true,
+            "name": "set-response-header"
+        }
+    ],
+    "id": "da8659e3-db85-435f-ba62-f95ec7615ddc",
+    "name": "my-filter-chain",
+    "route": null,
+    "service": {
+        "id": "992c4491-9962-48ff-8d5d-249ba82848d1"
+    },
+    "tags": null,
+    "updated_at": 1699230090
+}
+```
+
+### Notes
+
+As of this writing, only [Draft 4](https://json-schema.org/specification-links#draft-4)
+of the JSON schema specification is supported.
```

Comment:
Can you move this to the top of the doc instead, somewhere in the intro, and add it either as a prerequisite or in a note block like this:

```
{:.note}
> As of this writing, Kong only supports [Draft 4](https://json-schema.org/specification-links#draft-4)
of the JSON schema specification.